### PR TITLE
Hello Microverse!

### PR DESCRIPTION
**Pull request by Shewolf227**
In this exercise I implemented the following:

1. Created a new GitHub repo
2. Created a new branch
3. Added a `.gitignore` file and to this, I added the `test.md` file
4. Set up a linter for HTML and CSS
5. Added a `README` file
6. Added 3 files (each one in a separate commit):
    - index.html
    - styles.css
    - test.md

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (Deepakdanger) [inline .gitignore:1]: - [x] There is a .gitignore file listing `test.md` file and `node_modules`. Lets include `node_modules` also.

File: README.md [hunk 2]
```diff
@@ -1 +1,33 @@
...
+[Live Demo Link](https://SoniaKarungi.github.io/Hello-Microverse)
+
+
+## Getting Started
+
+
+
+To get a local copy up and running follow these simple example steps.
+
+- Clone this repository to your local machine
```
Comment (Deepakdanger) [inline README.md:24]: - [x] Let's add this section also, which contains the necessary code for running the project locally. For ex:-
```
git clone https://github.com/SoniaKarungi/Hello-Microverse.git
cd Hello-Microverse
```

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @SoniaKarungi, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (Shewolf227) [reply .gitignore:1]: Thanks for the feedback. I have implemented it all

**Review by CollinsTatang — APPROVED** [review]
Hi @SoniaKarungi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**